examples/petsc/area.c

```diff
@@ -147,7 +204,10 @@ int main(int argc, char **argv) {
     ierr = DMPlexCreateSphereMesh(PETSC_COMM_WORLD, topodim, simplex, &dm);
     CHKERRQ(ierr);
     // Set the object name
-    ierr = PetscObjectSetName((PetscObject) dm, "Cube"); CHKERRQ(ierr);
+    ierr = PetscObjectSetName((PetscObject) dm, "Sphere"); CHKERRQ(ierr);
+    if (strcmp(problemtype, "cube") == 0)
+      // Set the object name
+      ierr = PetscObjectSetName((PetscObject) dm, "Cube"); CHKERRQ(ierr);
```

Comment:
Could just `PetscObjectSetName((PetscObject)dm, problemtype)` for simpler code and simpler process to add a new manifold at some point (perhaps a torus)?

Reply:
Absolutely. Good idea!